napari-workshops/installation.md

```diff
@@ -98,53 +98,43 @@ as well and you can skip to the next section.
 2. We use an environment to encapsulate the Python tools used for this workshop.
    This ensures that the requirements for this workshop do not interfere with
    your other Python projects. To create the environment (named
-   `napari-tutorial`) and install Python 3.9 in it, enter the following command:
+   `napari-workshop`) and install Python 3.11 and napari in it, enter the following command:
 
     ```bash
-    conda create -n napari-tutorial python=3.9
+    conda create -n napari-workshop -c conda-forge python=3.11 napari pyqt jupyterlab
```

Comment:
I think it's easier to just install napari and jupyter directly, every workshop will need that I presume.